DayrSuryan/DSEthiop1.xml

```diff
@@ -672,15 +672,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <decoNote xml:id="b3" type="SewingStations">4</decoNote>
                           <decoNote xml:id="b4" type="Other">
                           <term key="leafStringMark"/>
-                         Leaf string markers (coloured threads) inserted in the upper part of <locus target="#1"/>,
-
-                             <locus target="#7"/>, <locus target="#13"/>, <locus target="#20"/>, <locus target="#29"/>,
-                             <locus target="#35"/>,
-                             <locus target="#48"/>, <locus target="#57"/>, <locus target="#63"/>, <locus target="#68"/>,
-                             <locus target="#78"/>, <locus target="#91"/>, <locus target="#96"/> , <locus target="#102"/>, <locus target="#112"/> (red),
-
-                             <locus target="#127"/>.
-                             A loose red thread is placed between <locus target="#20"/> and <locus target="#21"/>.
+                         Leaf string markers (coloured threads) inserted in the upper part of
+                         <locus target="#1 #7 #13 #20 #29 #35 #48 #57 #63 #68 #78 #91 #96 #102"/>,
+                         <locus target="#112"/> (red), and <locus target="#127"/>.
```

Comment:
I think it could be "...#102 #112 #127". Probably the information that one is red does not need to be kept?

Reply:
Pity, but I will remove it, otherwise it is uneven. One can figure out that it is red.

Reply:
Aha, if you like the colours they can all be restituted, it was orinigally like this:
```
 Leaf string markers (coloured threads) inserted in the upper part of <locus target="#1"/>  (yellow),
                             <locus target="#7"/> (red), <locus target="#13"/> (red), <locus target="#20"/> (yellow), <locus target="#29"/> (red), <locus target="#35"/> (yellow),
                             <locus target="#48"/> (yellow), <locus target="#57"/> (pale green), <locus target="#63"/> (pale green), <locus target="#68"/> (pale green),
                             <locus target="#78"/> (yellow), <locus target="#91"/> (red), <locus target="#96"/>  (red), <locus target="#102"/> (red), <locus target="#112"/> (red),
                             <locus target="#127"/> (pale green).
```
I deleted them for readability.

Reply:
Aha, ok, this is a different matter and changes the picture a little bit. If we have this, we can keep separate fols. I will restitute them, it is interesting.

Reply:
Like this? A kind of compromise.